vendor/github.com/ethereum/go-ethereum/whisper/whisperv6/whisper.go

```diff
@@ -378,6 +378,10 @@ func (whisper *Whisper) RequestHistoricMessages(peerID []byte, envelope *Envelop
 	return p2p.Send(p.ws, p2pRequestCode, envelope)
 }
 
+func (whisper *Whisper) SendHistoricMessageAck(peer *Peer, envelope *Envelope) error {
```

Comment:
I am not sure `ack` is a proper term here. It indicates that the communication is sync and the recipient is waiting actively for the acknowledge. `SendRequestCompleteCode` maybe? We have `p2pRequestCode` so it should make sense.

The could name would be `p2pRequestCompleteCode`.